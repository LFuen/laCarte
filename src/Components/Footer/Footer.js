import React, { Component } from "react";
import "./Footer.css";

class Footer extends Component {
  render() {
    return <footer className="foot">Â© LaCarte | 2020</footer>;
  }
}

export default Footer;
