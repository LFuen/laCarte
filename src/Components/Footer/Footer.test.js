import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import LaCarteContext from '../../context/LaCarteContext'
import Footer from './Footer'



describe(`Footer Component`, () => {
    describe(`Snapshot`, () => {
        it('renders the UI as expected', () => {
            const foot = renderer.create(<Footer />).toJSON()
            expect(foot).toMatchSnapshot()
        })
    })

    describe(`Smoke test`, () => {
        it(`Renders without crashing`, () => {
            
            const div = document.createElement('div')

            const value = {
                meals: 'string',
                chefs: 'string',
                cuisines: 'string',
                orders: 'string',
                addOrder: 'string',
                updateOrder: 'string',
                deleteOrder: 'string',
            };

            ReactDOM.render(
                    <LaCarteContext.Provider value={value}>
                        <Footer />
                    </LaCarteContext.Provider>,
                div)
            ReactDOM.unmountComponentAtNode(div)
        })
    })
})