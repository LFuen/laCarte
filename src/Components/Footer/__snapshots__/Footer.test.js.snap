// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Snapshot renders the UI as expected 1`] = `
<footer
  className="foot"
>
  Â© LaCarte | 2020
</footer>
`;
